ankerctl.py

```diff
@@ -138,6 +140,20 @@ def config_import(env, fd):
     install location
     """
 
+    # if fd is not provided, try to auto-detect
+    if fd is None:
+        useros = platform.system()
+
+        darfileloc = path.expanduser('~/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json')
+        winfileloc = path.expandvars(r'%LOCALAPPDATA%\Ankermake\AnkerMake_64bit_fp\login.json')
+
+        if useros == 'Darwin' and path.exists(darfileloc):
+            fd = open(darfileloc, 'r')
+        elif useros == 'Windows' and path.exists(winfileloc):
+            fd = open(winfileloc, 'r')
+        else:
+            exit("This platform does not support autodetection. Please specify file location with -f <filename>")
```

Comment:
Good stuff - I was working on a different solution, but I'd like to merge this for now :)

However, we don't use `-f` here. It would be `config import <filename>` instead

Reply:
I believe this was fixed in the latest commit.